docs/tutorials/friends_lobbies.md

```diff
@@ -0,0 +1,104 @@
+# Tutorials - Getting a list of lobbies friends are in
+
+In this quick tutorial we will cover how to get a dictionary of lobbies to friends that are visible to the user. This was created as it's a pretty common feature for multiplayer games.
+
+We will break down the different steam functions to call first before looking at the code samples.
+
+---
+
+## Functions explained
+
+```gdscript
+var number_of_friends: int = Steam.getFriendCount(Steam.FRIEND_FLAG_IMMEDIATE)
+```
+This must be ran before `Steam.getFriendByIndex` according to the Steamworks documentation, we'll be using this for our loop over each friend in our friends list.
+
+```gdscript
+var steam_id: int = Steam.getFriendByIndex(index, Steam.FRIEND_FLAG_IMMEDIATE)
+```
+Inside of the loop we'll call `Steam.getFriendByIndex` to get the Steam ID of our friend at this index.
+
+Note: you must use the same flag as the flag used inside of `Steam.getFriendCount`.
```

Comment:
I think a link to the flags would be a good idea here https://partner.steamgames.com/doc/api/ISteamFriends#EFriendFlags

Reply:
Good suggestion, I'm going to instead link to the Godotsteam docs for these flags: https://godotsteam.com/enums/friends/#friendflags